app/views/managementtools/noTeamUsersTable.scala.html

```diff
@@ -31,13 +32,13 @@
             <th id="@{usersTab.tabMessageKey}-role" class="bold w-0">Role</th>
         </tr>
 
-        @for((result, index) <- usersTab.searchResult.results.zipWithIndex) {
+        @for((result, index) <- usersTab.searchResult.zipWithIndex) {
 
                 <tr id="@{usersTab.tabMessageKey}-details-@index">
                     <td id="@{usersTab.tabMessageKey}-user-@index">
-                        <a href="">@result.name</a>
+                        <a href="@controllers.v2.routes.ManageUserController.displayUserDetails(result.id)">@result.name.getOrElse(result.id)</a>
```

Comment:
there is something like safeName in the Operator class that either displays the name of "PID X", don't know if maybe that could be used here?